stable/sealed-secrets/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+name: sealed-secrets
+description: A Helm chart for Sealed Secrets
+version: 0.1.0
+appVersion: 0.7.0
+kubeVersion: ">=1.8.0-0"
+home: https://github.com/bitnami-labs/sealed-secrets
+apiVersion: v1
```

Comment:
A chart needs to have maintainers. Since you're adding the chart, this should be you. Are you willing to maintain the chart and handle future issues and PRs for it?

Reply:
@olib963 I volunteer as a maintainer if you want, here is my info:

```yaml
maintainers:
  - name: stefanprodan
    email: stefan.prodan@gmail.com
```

You should also have a OWNERS file, here is an example: https://github.com/helm/charts/blob/master/stable/weave-cloud/OWNERS

Reply:
Thank you very much :D